src/main/java/org/example/Main.java

```diff
@@ -40,10 +38,10 @@ public static HttpServer startServer() {
      */
     public static void main(String[] args) throws IOException {
         final HttpServer server = startServer();
-        System.out.println(String.format("Jersey app started with WADL available at "
-                + "%sapplication.wadl\nHit enter to stop it...", BASE_URI));
-        System.in.read();
-        server.shutdown();
+        System.out.println(String.format("Jersey app started with WADL available at %sapplication.wadl\nHit enter to stop it...", BASE_URI));
+        while (true) {
```

Comment:
Not very elegant, but will be changed once development starts. 

Without it the server would just stop instantly when run with docker.

Reply:
Changed it back and added information to the Readme